wsd/COOLWSD.cpp

```diff
@@ -5094,6 +5139,7 @@ int COOLWSD::innerMain()
             = std::chrono::duration_cast<std::chrono::milliseconds>(timeNow - stampFetch);
         if (fetchUpdateCheck > std::chrono::milliseconds::zero() && fetchUpdateCheck >= durationFetch)
         {
+            processFetchUpdate();
```

Comment:
Do we do this on-start ? if fetchUpateCheck is non-zero?

Reply:
validation in case user disable